src/plugins/analysis/device_tree/test/test_device_tree.py

```diff
@@ -0,0 +1,36 @@
+from pathlib import Path
+
+from test.unit.analysis.analysis_plugin_test_class import AnalysisPluginTest
+from objects.file import FileObject
+from ..code.device_tree import AnalysisPlugin
+
+TEST_DATA = Path(__file__).parent.parent / 'test/data'
+
+
+class TestDeviceTree(AnalysisPluginTest):
+
+    PLUGIN_NAME = 'device_tree'
+
+    def setUp(self):
+        super().setUp()
+        config = self.init_basic_config()
+
+        self.analysis_plugin = AnalysisPlugin(self, config=config)
+        self.test_file = TEST_DATA / 'device_tree.dtb'
+
+    def tearDown(self):
+        super().tearDown()
+        # additional tearDown can go here
```

Comment:
This function can be removed, since it does nothing more than call the function of the parent class